libs/payments/stripe/src/lib/factories/invoice.factory.ts

```diff
@@ -100,9 +102,9 @@ export const StripeInvoiceFactory = (
   tax: faker.number.int({ max: 1000 }),
   test_clock: null,
   total: faker.number.int({ max: 1000 }),
-  total_discount_amounts: null,
+  total_discount_amounts: [StripeTotalDiscountAmountsFactory()],
```

Comment:
Should these changes be reverted? Typically for nullable or empty array fields, if they need to be provided, the `override` should be used.